user-manual/transfer/import-metadata.rst

```diff
@@ -54,43 +62,46 @@ MDTYPE="OTHER". A sample of the METS output is available below.
 2. For compound objects, create one or more subdirectories in the
    objects directory, each containing the files that form a compound object.
 
+.. figure:: images/compound-csv-view.*
+   :align: center
+   :alt: Metadata folder in transfer directory with compound objects
+
+   Compound object file structure.
+
 .. important::
 
    The subdirectory names must not contain spaces or other forbidden characters.
 
-3. Add a csv file to the metadata folder for the transfer called
+3. Add a CSV file to the metadata folder for the transfer called
    ``metadata.csv``.
 
-   * The first row of the csv file consists of field names. Field names must not
+   * The first row of the CSV file consists of field names. Field names must not
      include spaces.
 
    * Dublin Core field names must contain the "dc" element in the name, e.g.
-     "dc.title". Note that the Dublin Core is not validated - this is up to the
+     "dc.title". Note that the Dublin Core is not validated -- this is up to the
      user.
 
    * Dublin Core terms must contain "dcterms" in the name, e.g.
-     "dcterms:abstract". As above, the Dublin Core is not validated - this is up
-     to the user.
+     "dcterms:abstract". As above, the Dublin Core is not validated -- this is 
+     up to the user.
 
    * Each subsequent row contains the field values for a single directory or
      file.
 
-   * As of version 1.4, both directory and object level metadata is allowed
-     in the metadata.csv. The csv can contain only object level, only directory
-     level, or a combination of both.
-
    * For multi-value fields (such as dc.subject), the entire column is repeated
      and each column contains a single value (i.e. there should be multiple
      dc.subject columns if there are multiple subject terms).
 
    * Empty columns can be deleted, if you prefer.
 
-   * The first column in the metadata.csv file must be a "filename" column
-     listing the filepath and filename or directory name of each object or
-     directory: e.g. "objects/BrocktonOval.jp2" , or "objects/Jan021964".
+   * The first column in the ``metadata.csv`` file must be a "filename" column.
+     This column should list the filepath and filename (e.g. 
+     "objects/BrocktonOval.jp2") or directory name of each object or
+     directory (e.g. "objects/Jan021964").
 
-   * If you have directory level metadata fill out the fields on the same line
-     as the directory (ie objects/)
+   * If you have directory level metadata, fill out the fields on the same line
+     as the directory (e.g. objects/).
```

Comment:
Maybe there should there be an admonition somewhere about the fact that you need to include ``objects/`` in the path even if the materials aren't in an objects subfolder? It's a legacy artifact from the days when all digital objects in the transfer had to be nested in such a way.

Reply:
I agree with this!!! I thought about that and forgot to add it in.